src/stub.c

```diff
@@ -23,6 +23,14 @@ const BYTE Signature[] = { 0x41, 0xb6, 0xba, 0x4e };
 #define OP_CREATE_INST_DIRECTORY 8
 #define OP_MAX 9
 
+/* Manages digital signatures */
+#define SECURITY_ENTRY(header) (header->OptionalHeader.DataDirectory[IMAGE_DIRECTORY_ENTRY_SECURITY])
```

Comment:
Is this "optional header" always present or do we need to check that it's present?

Reply:
@jakedouglas it's not optional for PE (exe files), but is optional for some other binary formats that windows uses, so it's safe for us to just access it here